_posts/2023-04-20-solvationJOSS.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: SolvationAnalysis Published in JOSS
+---
+
+We're happy to announce that the Python package, [`SolvationAnalysis`][solvation], is now [published][paper] in the [Journal of Open Source Software][joss] (JOSS). Originally developed by @orioncohen as a [GSoC 2021 project][gsocblog], `SolvationAnalysis` builds on `MDAnalysis` and `pandas` to automate extraction of solvation information from molecular dynamic simulations in order to compute key solvation properties. To get started with using `SolvationAnalysis` in your workflow, check out the documentation and tutorials on [readthedocs][docs].
+
```

Comment:
> We're happy to announce that the Python package, [`SolvationAnalysis`][solvation], is now [published][paper] in the [Journal of Open Source Software][joss] (JOSS). Originally developed by @orioncohen as a [GSoC 2021 project][gsocblog], `SolvationAnalysis` builds on `MDAnalysis` and `pandas` to automate extraction of solvation information from molecular dynamic simulations in order to compute key solvation properties. To get started with using `SolvationAnalysis` in your workflow, check out the documentation and tutorials on [readthedocs][docs].

We're happy to announce that the Python package, [`SolvationAnalysis`][solvation], is now [published][paper] in the [Journal of Open Source Software][joss] (JOSS). Originally developed by @orioncohen as a [GSoC 2021 project][gsocblog], `SolvationAnalysis` builds on `MDAnalysis` and `pandas` to make analyzing solvation structure much, much easier. With a few lines of code, you can extract key solvation properties from any molecular dynamics simulation. With a few more, you can visualize solvation trends within and between different solutions. Further, SolvationAnalysis is designed with extensibility in mind, exposing a core representation of solvation that users can use for brand new analyses. To get started with using `SolvationAnalysis` in your workflow, check out the documentation and tutorials on [readthedocs][docs].

^^ Couple extra lines describing use, feel free to edit!